dot/network/host.go

```diff
@@ -143,14 +142,29 @@ func newHost(ctx context.Context, cfg *Config) (*host, error) {
 	// wrap host and DHT service with routed host
 	h = rhost.Wrap(h, dht)
 
+	cacheSize := 64 << 20 // 64 MB
+	config := ristretto.Config{
+		NumCounters: int64(float64(cacheSize) * 0.05 * 2),
+		MaxCost:     int64(float64(cacheSize) * 0.95),
+		BufferItems: 64,
```

Comment:
what do these values do?

Reply:
Good question.

Reply:
These params are used to keep the cache size fixed and helps in eviction.